src/components/modules/TagPostsOverlapTitle/TitleRow.tsx

```diff
@@ -78,7 +78,7 @@ export default TitleRow;
 
 const styles = StyleSheet.create({
   titleRow: {
-    width: '100%',
+    width: '300px',
```

Comment:
The problem with putting the width here is that it also affects the ad unit. What do you think about putting it in the style attribute of the ModuleTitle component?  (line 64)